### PR TITLE
The fourth large cleanup of Base

### PR DESCRIPTION
**Pull request by Yuki2718**
```
! ---------- Typo (double comma)
||adv.wp.pl/RM/Box/*.mp4$media,redirect=noopmp4-1s,,important


! ---------- In EL
||fryawlauk.com^
||g--o.info^$third-party

/poppy^$script,domain=upornia.com|hclips.com

youtube.com##ytd-promoted-sparkles-web-renderer


! ---------- Redundant
! @@||ajax.googleapis.com/ajax/libs/jquery/
@@||ajax.googleapis.com/ajax/libs/jquery/$domain=2ddl.unblocked.vc|daporn.com|imagefap.com|linkdrop.net|gelbooru.com|fmovies.is|fmovies.se|imagespicy.site|mycloud.to

! @@||cdnjs.cloudflare.com/ajax/libs/$domain=fmovies.is|fmovies.se|fmovies.to
!+ PLATFORM(ios, ext_android_cb)
@@||cdnjs.cloudflare.com/ajax/libs/$domain=fmovies.is|fmovies.se

! @@||adm.fwmrm.net/p/msn_live/AdManager.swf$domain=msn.com|akamaized.net
@@||adm.fwmrm.net/p/msn_live/AdManager.swf$domain=msn.com

! @@||hitomi.la/jquery.
!+ NOT_OPTIMIZED PLATFORM(ios, ext_safari)
@@||hitomi.la/jquery.min.js

! @@||cdn.staticdata.site/js/loadjs*.min.js$domain=strikeout.nu
@@||cdn.staticdata.site/js/loadjs.min.js^$domain=strikeout.nu


@@||iconmonstr.com^$generichide
@@||pagead2.googlesyndication.com/pagead/js/adsbygoogle.js$domain=audiotag.info
@@||ad.a-ads.com/*?size=$domain=getitall.top
@@||de-baystars.doorblog.jp^$generichide
@@||gametter.com^$generichide
! @@||finn.no/*ad.html?
@@||finn.no/*/ad.html?
! @@||googletagservices.com/tag/js/gpt.js
@@||googletagservices.com/tag/js/gpt.js$domain=hdblog.it|hdmotori.it
@@||googletagservices.com/tag/js/gpt.js$domain=boston.com|calgaryherald.com|calgarysun.com|celebuzz.com|infinitiev.com|montrealgazette.com|saveur.com
@@||googletagservices.com/tag/js/gpt.js$domain=joe.ie|joe.co.uk
@@||googletagservices.com/tag/js/gpt.js$domain=nfl.com

! ---------- Dead domain (foreign)

ddd.jandan.net

! http://www.bysp.pw/?m=vod-type-id-1-pg-85.html
bysp.pw

! m.uerzyr.cn
uerzyr.cn

! http://sl.ce22d.cn/360/index.html
sl.ce22d.cn

lulu.dian500.com.cn
bobo.xmwty.com
hueren.us

! http://www.10co.org/
10co.org

cevd0tnyvecontent.net

! http://gooby.naurunappula.com/screen/c1/60/c160dce6ede49bda/0/1165383.jpg
gooby.naurunappula.com

! http://www.all-stadium.net/
all-stadium.net

! http://www.playerhd2.pw/jwplayer5/x-fonter.gif
playerhd2.pw

! https://ganool.ltd
ganool.ltd

! http://cfs1.uzone.id/2fn7a2/request
cfs1.uzone.id

jor-el.net
cloudboost.co.il
treemedia.co.il
goalim4you.site
msp.ticdn.it

! http://www.guardarefilm.live/streaming-film/12093-ascensione-2017.html
guardarefilm.live

! https://jdownlader2premiums.blogspot.de/
jdownlader2premiums.blogspot.de

! https://iu.copytoon57.com/5sekjwmb.jpg
iu.copytoon57.com

! https://partner.toptoon.com/
partner.toptoon.com

freestory01.net

! https://ttonga.net/bbs/content.php?co_id=webtoon_gate&gate=toon1004
ttonga.net

! http://torrentleea.com/
torrentleea.com

26ariamovie.in

! https://anime-ro-sub.ro/
anime-ro-sub.ro

! http://get-contents.video-ads.tk/files/
get-contents.video-ads.tk

! http://jh333.pw/
jh333.pw

allercdn.net
ad.52av.tv

! http://uniad.phimmoi.net/publisher/go/
uniad.phimmoi.net

! https://www.vietmod.pro/
vietmod.pro

n.tivi8k.net

! https://www.tonystream.com/wp-includes/js/min.js
tonystream.com

! http://jav8.net/category/sex-loan-luan/
jav8.net
```

### CODE REVIEW

**Comment by Yuki2718** [comment]
Please also check/remove:
```
! ---------- Dead domain (general_extensions)
5278.live
fuckvideos.adult
fulltube.porn
imgic.ru
inpixme.pw
lucotube.com
revolutions.pw
muskaer.com
postpics.xyz
2ddl.ag
flashxx.tv
uplod.so
4gmovies.mobi
asianpornempire.stream
bdsmpornhd.com
bigohot.zone
bokepseks.tube
ezsports.stream
ghonche.xyz
incestoreal.net
iptvsatlinks.blogspot.com
jkanime.site
knaben.tk
kobestreams.ch
link-protect.me
lokervids.win
megafilmestorrent.cc
netpics.top
onlinefilmsitesii.net
propix.pw
prxy.party
rumahbokep.fun
todaypk.lol
videobokepz.men
videobokepz.stream
videocrot.city
world4freeus.org
xfilmywap.me
cutadlink.net
moviezwaphd.la
9stream.pw
fbbcdn.xyz
hentaistream.video
imp3.live
extreme-down.im
flashx.bz
leakedearly.gdn
peliculasveo.gratis
xmovies8.im
soundpark.pt
rarbg.unblocked.work
4553t5pugtt1qslvsnmpc0tpfz5fo.xyz
ukpirate.click
streamin.to
analstreams.net
bioskop378.tv
dermclarcis.ru
dramafree.se
eroticporn.co
estrenosdtl.tv
hdvideos.porn
hugeblackcocks.org
lesbian-tube.net
livesx.ru
mdhstream.net
p2p4u.ru
sambapornobrasileiro.com
teenvideos.download
toppornstars.org
openload.cz
estream.nu
ahoypirate.click
baytpbportal.fi
piratebay.click
pirateproxy.click
ahoypirate.in
bayception.org
bayhypertpb.be
pirata.pw
piratebay.town
grandxxi.win
indomovies88.biz
japscan.co
spashopaz.com
360nobs.org
adsviv.com
almidan9.net
downlaodmovies.com
downloadmp3song.live
fridaymovieshd.com
fullxxxonline.com
hizlifullfilmizle.com
kickasstorrent.com.pl
marstv.pw
mktba22.blogspot.com
moviescounter.cool
mp3-search.click
mp3-search.top
mp3skulls.today
musicpleer.es
onlinemovierasiganyogis.in
onlinemoviesrasiganyogis.in
onlinemoviezrasiganyogi.com
onlinemovizrasiganyogi.in
starmusiq.la
tambayanlivetv.com
tnhd.co
watchcommunity.cc
watchtvseries.unblckd.cx
allinternetnews.com
allvids.co
newsbaxs.com
thegroupmusic.com
hallan.club
! parked
gowatchfreemovies.to
cat3movie.us
mp3xyz.co
retrotube.porn
hitbottlewood.com
welcomehula.com
sportstream2.pw
stream.freelive365.com
! parked
zembed.to
learningfromtrip.info
jibfly.com
phillip-mellody.me
thevidhd.club
myfeed2all.eu
shassets.pw
torrents24.top
torrentz-eu.top
torrentz5.net
xxxmilfporn.xxx
streamsk.com
catcuturl.tk
vrlesbian.video
gogoanime.to
flashx.sx
24scor.com
gaytoday.xxx
putlocker.cz
pornovo.cm
birdenplay.com
! NOT pornovo.com
pornovo.cm
birdenplay.com
apkpromod.net
propellerthevideo.me
zoomtv.me
aggregatorgetb.com
castfree.me
download.jkmk.net

! ---------- Obsolete
! general_extensions
hentaisea.com#%#//scriptlet('abort-current-inline-script', 'document.querySelectorAll', 'popMagic')
thegay.com#%#//scriptlet("abort-on-property-read", "vertis.popunder.neverblock")
thegay.com#%#//scriptlet("set-constant", "vdaVars.neverblock.loaderUrl", "undefined")
vidstream.*$$script[tag-content="break;case $."][min-length="30000"][max-length="85000"]
vidstream.*#%#//scriptlet("abort-current-inline-script", "String.fromCharCode", "/emZnbG9hZGVk|break;case \$\./")
vidstream.*#%#//scriptlet("abort-on-property-write", "GLX_GLOBAL_UUID_RESULT")
ytmp3.cc$$script[tag-content="zfgloaded"][min-length="30000"][max-length="80000"]
ytmp3.cc#%#//scriptlet("abort-on-property-read", "zfgformats")
```

File: EnglishFilter/sections/content_blocker.txt [hunk 1]
```diff
@@ -1196,8 +1194,6 @@ topbusiness24.ru,mydomsam.ru,hubuhu.ru,moysadinfo.ru,mybusinessplus.ru,dukand.ru
 @@||widgets.outbrain.com/images/$image,domain=afreesms.com
 ! https://github.com/AdguardTeam/AdguardFilters/issues/68913
 @@||dongknows.com^$generichide
-! https://github.com/AdguardTeam/AdguardFilters/issues/68815
-@@||ad.a-ads.com/*?size=$domain=getitall.top
```
Comment (AdamWr) [inline EnglishFilter/sections/content_blocker.txt:1200]: It seems that I added a rule with wrong domain in this commit - https://github.com/AdguardTeam/AdguardFilters/commit/26b7d0337e2240f16f594526b069b78d34f63949#diff-5d33fa79f5277920586720bffd6c91c26924c4cbeedfc0907448ac69e6b6e656R1192

I will fix it.

**Review by AdamWr — APPROVED** [review]

**Comment by AdamWr** [comment]
I will try to check domains from this comment - https://github.com/AdguardTeam/AdguardFilters/pull/70462#issuecomment-748624245 too.

Thank you.

**Comment by AdamWr** [comment]
>I will try to check domains from this comment - #70462 (comment) too.

Done.

**Comment by Yuki2718** [comment]
Thx!

**Pull request merged**